v2/options.go

```diff
@@ -46,8 +46,10 @@ type ReadWriteOption interface {
 // a zero-length section as the end of the input CAR file. For example, this can
 // be useful to allow "null padding" after a CARv1 without knowing where the
 // padding begins.
-func ZeroLengthSectionAsEOF(o *ReadOptions) {
-	o.ZeroLengthSectionAsEOF = true
+func ZeroLengthSectionAsEOF(enable bool) ReadOption {
+	return func(o *ReadOptions) {
+		o.ZeroLengthSectionAsEOF = true
```

Comment:
Thinking out-loud are there any other options in code that we want to make configurable like this before the release tag?

Reply:
I didn't find any. I'm honestly not too bothered about boolean vs "just let the user set the non-default", but either way, I wanted it to be consistent.